data_scrapers/san_francisco.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+import json
+from typing import Dict, List
+from collections import Counter
+from data_scrapers.utils import get_data_model, SocrataApi
+
+def get_county() -> Dict:
+    """ Main method for populating county data.json """
+    # TODO: Add deaths data when it is available through the api.
+    # TODO: In get_race_eth_table(), update code when SF provides a race label for the Native American datapoint.
+
+    # Load data model template into a local dictionary called 'out'.
+    out = get_data_model()
+    # create a SocrataApi instance
+    RESOURCE_IDS = {'cases_deaths_transmission': 'tvq9-ec9w', 'age_gender': 'sunc-2t3k',
+                     'race_eth': 'vqqm-nsqg', 'tests': 'nfpa-mg4g'}
+    session = SocrataApi('https://data.sfgov.org/', RESOURCE_IDS)
```

Comment:
This setup works fine, but feels a bit weird to me. If I’m reading right, you’re really just putting a `resource_ids` attribute on `SocrataApi` so you can access it in other places here, and it actually has nothing to do with `SocrataApi` itself.

Reply:
Yes. I can see now why it's weird. Should I take the `resource_ids` off of the `SocrataApi` instance? I can just as easily pass them around along with the `session`.

Reply:
Ok, I ended up taking it off  the `SocrataApi` because it is way easier to do type hints. I begrudgingly admit that this type hinting thing might be forcing me to think more rigorously.

Reply:
Sorry I wasn’t able to reply yesterday, but this seems good! 😅